torchinductor/lowering.py

```diff
@@ -726,6 +773,7 @@ def _constant_like(x, *, dtype=None, device=None, layout=0, pin_memory=False):
     return _constant_like
 
 
+empty_like = register_lowering(aten.empty_like)(constant_like(0))
```

Comment:
I think there is a perf difference between empty (non-initialized) and zeros (initialized)?

Reply:
TorchInductor doesn't actually realize the zero tensor unless users return uninitialized memory.  So in the common case the zero initializer will be optimized away.